tests/base.py

```diff
@@ -170,6 +170,20 @@ def expected_metadata(self):  # DOCS_BUG https://stitchdata.atlassian.net/browse
                 self.REPLICATION_KEYS: {"updatedAt"},
                 self.EXPECTED_PAGE_SIZE: 100,
                 self.OBEYS_START_DATE: True
+            },
+            "custom_object_campaigns": {
+                self.PRIMARY_KEYS: {"id"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"updatedAt"},
+                self.EXPECTED_PAGE_SIZE: 100,
+                self.OBEYS_START_DATE: True
+            },
+            "custom_object_contacts": {
```

Comment:
How about testing the corner case scenario for the stream name like - `custom_object_custom_object_contacts`?

Reply:
We only rename custom objects stream if their original name matches with standard stream names like `contacts`, `campaigns`, `deals`. Therefore, if custom object is named as `custom_object_custom_object_contacts` then we won't take any action on it.